### ISSUE TITLE
Rscript process command error on Windows

### ISSUE THREAD

**Issue by j-pillon** [campsych/concerto-platform#26]
Hi

How to reproduce:
- OS: Windows 7 sp1
- Use Firefox 45.2 ESR or Chrome 52 as web browser
- Create a questionnaire (e.g. tutorial "How to create a questionnaire test")
- Click "Debug test"

Results:
- In the new test tab : "Error encountered. Execution halted."
- In JS console (in french for R messages) :
```
Le chargement a nécessité le package : concerto5
Le chargement a nécessité le package : session
Le chargement a nécessité le package : rjson
Error in fromJSON(commandArgs(TRUE)[2]) : 
  unexpected character "i"; expecting opening string quote (") for key value
  Exécution arrétée
```

Analyse:
- i get log concerto.log and concerto.log.r in "\src\Concerto\TestBundle\Resources\local\<hash_session>" and test rscript command line (without stdout redirection).
- it seems json arguments aren't correct because of double quotes missing.
- So, escape characters need to be escaped too...

Solution:
- Update "\src\Concerto\TestBundle\Command\StartProcessCommand.php" for "not Linux" rscript (line 281)
instead of
```
$cmd = "\"" . $rscript_exec . "\" --no-save --no-restore --quiet \"$ini_path\" \"" . addcslashes($test_server_connection, '"') . "\" \"" . addcslashes($r_server, '"') . "\" \"" . addcslashes($submitter, '"') . "\" \"" . addcslashes($client, '"') . "\" $test_session_id \"$wd\" \"$pd\" $murl \"" . addcslashes($values, '"') . "\" >> \"" . $this->logPath . "\" > \"" . $this->rLogPath . "\" 2>&1";
```
write
```
$cmd = "\"" . $rscript_exec . "\" --no-save --no-restore --quiet \"$ini_path\" " .
	"\"" . addcslashes(addcslashes($test_server_connection, '"'), '"\\') . "\" ".
	"\"" . addcslashes(addcslashes($r_server, '"'), '"\\') . "\" ".
	"\"" . addcslashes(addcslashes($submitter, '"'), '"\\') . "\" ".
	"\"" . addcslashes(addcslashes($client, '"'), '"\\') . 
	"\" $test_session_id \"$wd\" \"$pd\" $murl " .
	"\"" . addcslashes($values, '"') . "\" " .
	">> \"" . $this->logPath . "\" > \"" . $this->rLogPath . "\" 2>&1";
```

bye

**Comment by przemyslawlis** [comment]
Can you please paste contents of your app/config/parameters_nodes.yml file?

**Comment by j-pillon** [comment]
No change in my app/config/parameters_nodes.yml file, i.e.:

```
parameters:
    nodes:
        -
            id: local
            protocol: http
            host: localhost
            dir: /
            connection: local
            hash: someRandomHash
```

**Comment by przemyslawlis** [comment]
Should be fixed in: cad8e9120b0ff79dcc1b5ee53f40413f63221dc3
